<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>目标用户名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineDestName"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="TextRead">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="TextWrite"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonSend">
       <property name="text">
        <string>发送消息</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
